postbuild.sh

```diff
@@ -0,0 +1,4 @@
+# Temporary solution to serve app under `dsp.maltablock.org/liquiairdrops`.
+# Remove it once it will be fixed on server side (in nginx configs).
+# (all subpaths should be served with root index.html).
+cd build && mkdir liquiairdrops && cp index.html liquiairdrops/
```

Comment:
oh, that's smart 🤣

You only need to copy the index.html and not the JS source files because the index.html has the %PUBLIC_URL, so it will always take the ones from the roto folder, right?

Reply:
right :)